dali/operators/random/normal_distribution_gpu.cu

```diff
@@ -36,35 +37,33 @@ class NormalDistributionGPU : public NormalDistribution<GPUBackend, NormalDistri
   explicit NormalDistributionGPU(const OpSpec &spec)
       : NormalDistribution<GPUBackend, NormalDistributionGPU>(spec) {
     assert(max_batch_size_ < backend_data_.max_blocks_);
-    dists_gpu_ = kernels::memory::alloc_unique<uint8_t>(
-        kernels::AllocType::GPU, kDistMaxSize * max_batch_size_);
-    dists_cpu_ = kernels::memory::alloc_unique<uint8_t>(
-        kernels::AllocType::Pinned, kDistMaxSize * max_batch_size_);
+    dists_cpu_.reserve(kDistMaxSize * max_batch_size_);
+    dists_gpu_.resize(kDistMaxSize * max_batch_size_);
+    dists_gpu_.clear();
```

Comment:
How about adding `reserve` to DeviceBuffer?

Reply:
will do